openquake/qa_tests_data/classical/case_13/job.ini

```diff
@@ -3,6 +3,7 @@
 description = Classical PSHA QA test
 calculation_mode = classical
 random_seed = 23
+split_sources = false
```

Comment:
Maybe you can add a short  comment on this parameter

Reply:
It is a parameter introduced years ago used to avoid splitting the sources (in this test it reduces the number of spawned tasks and makes the test a bit faster).